scripts/gke/util.sh

```diff
@@ -246,7 +247,7 @@ gcpKsApply() {
   createKsEnv
 
   if [[ -z $DEFAULT_KUBEFLOW_COMPONENTS ]]; then
-    export KUBEFLOW_COMPONENTS+=',"cloud-endpoints","cert-manager","iap-ingress"'
+    export KUBEFLOW_COMPONENTS+=',"cloud-endpoints","cert-manager","iap-ingress", "proxy-agent"'
```

Comment:
If use ```proxy-agent```, then ```cloud-endpoints","cert-manager","iap-ingress``` is not needed.
Can we control by parameter?
Or we can just implement the logic in kfctl-go which should be easier